### PR TITLE
bug 1987230: Operators should not create watch channels very often: bump apirequests upperbounds in 4.9

### PR DESCRIPTION
**Pull request by ingvagabund**

### CODE REVIEW

File: test/extended/apiserver/api_requests.go [hunk 1]
```diff
@@ -126,9 +126,9 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 				"ingress-operator":                       371,
 				"kube-apiserver-operator":                260,
 				"kube-controller-manager-operator":       145,
-				"kube-storage-version-migrator-operator": 36,
+				"kube-storage-version-migrator-operator": 58,
```
Comment (deads2k) [inline test/extended/apiserver/api_requests.go:129]: wow, I wonder what we did.

File: test/extended/apiserver/api_requests.go [hunk 2]
```diff
@@ -205,10 +205,10 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 				"console-operator":                       154,
 				"csi-snapshot-controller-operator":       59,
 				"dns-operator":                           59,
-				"etcd-operator":                          162,
+				"etcd-operator":                          191,
```
Comment (deads2k) [inline test/extended/apiserver/api_requests.go:208]: wow, this is now more than 50% higher than other platforms..

File: test/extended/apiserver/api_requests.go [hunk 3]
```diff
@@ -137,29 +137,29 @@ var _ = g.Describe("[sig-arch][Late]", func() {
...
+				"kube-storage-version-migrator-operator": 33,
+				"machine-api-operator":                   31,
+				"marketplace-operator":                   11,
 				"openshift-apiserver-operator":           215,
-				"openshift-config-operator":              37,
-				"openshift-controller-manager-operator":  110,
-				"openshift-kube-scheduler-operator":      102,
+				"openshift-config-operator":              39,
+				"openshift-controller-manager-operator":  124,
+				"openshift-kube-scheduler-operator":      116,
```
Comment (soltysh) [inline test/extended/apiserver/api_requests.go:162]: 10% for both kcm-o and ks-o would be nice to check why is that

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**